### PR TITLE
added category to components, updated metadata to new data format

### PR DESCRIPTION
**Pull request by norlandrhagen**

### CODE REVIEW

File: data/EW/metadata.json [hunk 1]
```diff
@@ -1,5 +1,70 @@
 {
-  "contributors": [],
+  "contributors": [
+    {
+      "Type": "CDR Company",
+      "Name": "Eion",
+      "Affiliation": "",
+      "initial": "1.0",
+      "notes": "",
+      "enhanced-weathering": "1.0, 1.1"
```
Comment (katamartin) [inline data/EW/metadata.json:9]: Would it be possible to make a couple of tweaks here?

- consistently use lowercase keys
- roll up `initial` and `enhanced-weathering` into a single `versions` array

```suggestion
      "type": "CDR Company",
      "name": "Eion",
      "affiliation": "",
      "versions": ["1.0", "1.1"],
      "notes": "",
```

File: data/DAC/metadata.json [hunk 2]
```diff
@@ -1,5 +1,30 @@
 {
-  "contributors": [],
+  "contributors": [
+    {
+      "Type": "cdr company",
```
Comment (katamartin) [inline data/DAC/metadata.json:4]: The uppercased **values** were fine, but it'd be great to normalize the **keys** to be lowercased!

Comment (norlandrhagen) [reply data/DAC/metadata.json:4]: Oh haha, good catch! Will revert and update.

**Review by katamartin — APPROVED** [review]
looks great to me!

**Pull request merged**